schemas/bridging/broadcastRequest.schema.json

```
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/broadcastRequest.schema.json",
+  "title": "BroadcastRequest",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeRequest.schema.json"
+    },
+    {
+      "type":"object",
+      "properties": {
+        "type": {
+          "const": "broadcastRequest"
+        },
+        "payload": {
+          "type": "object",
+          "properties": {
+            "channel": {
+              "$ref": "../api/channel.schema.json"
+            },
+            "context": {
+              "$ref": "../context/context.schema.json"
+            }
+          },
+          "additionalProperties": false,
+          "required": ["channel", "context"]
+        },
+        "meta": {
+          "properties": {
+            "source": {
+              "$ref": "../api/appIdentifier.schema.json"
+            }
+          }
+        }
```

```suggestion
        "meta": {
          "properties": {
            "requestGuid": true,
            "timestamp": true,
            "destination": true,
            "source": {
              "$ref": "../api/appIdentifier.schema.json"
            }
          },
          "additionalProperties": false
        }
```

I think this might work - needs to redeclare all the properties from the composed schema as allowed...